reproducibility_project/src/engines/gomc/project.py

```diff
@@ -1,23 +1,2174 @@
+                # need to add equilb_design_ensemble_number by 1 so it is fixed to run the correct job
+                # so it is rerun if restarted
+                job.doc.equilb_design_ensemble_number += 1
+
+
+# ******************************************************
+# ******************************************************
+# equilb NPT or GEMC-NVT - starting the GOMC simulation (end)
+# ******************************************************
+# ******************************************************
+
+# ******************************************************
+# ******************************************************
+# production run - starting the GOMC simulation (start)
+# ******************************************************
+# ******************************************************
+@Project.pre(part_4c_job_equilb_design_ensemble_completed_properly)
+@Project.pre(part_2d_production_control_file_written)
+@Project.pre(pymbar_stabilized_equilb_design_ensemble)
+@Project.post(part_3d_output_production_run_started)
+@Project.post(part_4d_job_production_run_completed_properly)
+@Project.operation.with_directives(
+    {
+        "np": lambda job: ff_info_dict.get(job.sp.forcefield_name).get("ncpu"),
+        "ngpu": lambda job: ff_info_dict.get(job.sp.forcefield_name).get(
+            "ngpu", 0
+        ),
+        "memory": memory_needed,
+        "walltime": walltime_gomc_hr,
+    }
+)
+@flow.with_job
+@flow.cmd
+def run_production_run_gomc_command(job):
+    """Run the gomc production_run simulation."""
+    print("#**********************")
+    print("# Started the run_production_run_gomc_command function.")
+    print("#**********************")
+
+    control_file_name_str = job.doc.production_run_ensemble_dict[
+        str(job.doc.equilb_design_ensemble_number)
+    ]["input_name_control_file_name"]
+
+    output_file_name_str = job.doc.production_run_ensemble_dict[
+        str(job.doc.equilb_design_ensemble_number)
+    ]["output_name_control_file_name"]
+
+    print(f"Running simulation job id {job}")
+    run_command = "{}/{} +p{} {}.conf > out_{}.dat".format(
+        str(job.doc.gomc_binary_path),
+        str(job.doc.production_ensemble_gomc_binary_file),
+        str(ff_info_dict.get(job.sp.forcefield_name).get("ncpu")),
+        str(control_file_name_str),
+        str(output_file_name_str),
+    )
+
+    return run_command
+
+
+# ******************************************************
+# ******************************************************
+# production run - starting the GOMC simulation (end)
+# ******************************************************
+# ******************************************************
+
+# ******************************************************
+# ******************************************************
+# signac end code (start)
+# ******************************************************
+# ******************************************************
 if __name__ == "__main__":
     pr = Project()
```

Comment:
Since we are running this file itself, we will always be accessing through this `main` method.

So you do not need the code above that looks like this:

```py3

# define project
project = signac.init_project("mosdef_reproducibility")
```

Reply:
removed